inference-engine/src/transformations/include/transformations/mul_add_squence_fusion.hpp

```diff
@@ -0,0 +1,150 @@
+// Copyright (C) 2018-2020 Intel Corporation
```

Comment:
Smth wrong with this file. It was removed or redesigned.

Reply:
done, thanks: https://github.com/openvinotoolkit/openvino/pull/2671/commits/bdf93e1fe34e278b42177724af50d51ddcbb5884